script/git_tool.py

```diff
@@ -252,6 +254,17 @@ def get_repo_info_manifest_xml(self, repo_path: str = "./",
             lst_project = [lst_project]
         dct_remote = {a.get("@name"): a.get("@fetch") for a in lst_remote}
         for project in lst_project:
+            groups = project.get("@groups")
+            lst_group = groups.split(',') if groups else []
+            # Continue if lst_filter exist and group in filter
+            for group in lst_group:
+                if lst_filter_group and group not in lst_filter_group:
+                    continue
+                else:
+                    break
+            else:
```

Comment:
I can't find the "if" matching with this "else"

Reply:
It's a "for" "else", use when not meet a break.